%YAML:1.0
model_type: PINHOLE
camera_name: onborad_cam_davis
image_width: 640
image_height: 480
distortion_parameters:
   k1: -4.6057485804908826e-01
   k2: 5.1615329117730113e-01
   p1: -4.6442051501698875e-05
   p2: 3.4318297804546210e-03
projection_parameters:
   fx: 5.3633325932983780e+02
   fy: 5.3631797700847164e+02
   cx: 3.2090009280822994e+02
   cy: 2.3404853514480661e+02